packages/istio/charts/1.6.500/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
```

Comment:
Need to check long term if we can switch to Role/RoleBinding in istio-system NS only.

Reply:
per design discussion 07/17 this will remain clusterrole binding